templates/zerver/integrations.html

```diff
@@ -1639,7 +1639,8 @@
           <p>To configure Zulip notifications for a particular Redmine project,
             visit the project's Settings page.</p>
           <p>
-            In either case, fill out the bot email address and API key, and the
+            In either case, fill out zulip server by <code>{{ external_api_path_subdomain }}</code>,
```

Comment:
Yeah but the redmine plugin isn't using the generic Python client -- it's written in Ruby: https://github.com/zulip/zulip-redmine-plugin